core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -769,6 +789,36 @@ public function getModules()
 		return $arrModules;
 	}
 
+	/**
+	 * Return all content elements
+	 *
+	 * @return array
+	 */
+	public function getContentElements()
+	{
+		$groups = array();
+
+		foreach ($GLOBALS['TL_CTE'] as $k=>$v)
+		{
+			foreach (array_keys($v) as $kk)
+			{
+				$groups[$k][] = $kk;
+			}
+		}
+
+		return $groups;
+	}
+
+	/**
+	 * Return all form fields
+	 *
+	 * @return array
+	 */
+	public function getFormFields()
+	{
+		return array_keys($GLOBALS['TL_FFL']);
```

Comment:
you _could_ save this method by just using `'options' => array_keys($GLOBALS['TL_FFL'])` inside the DCA

Reply:
Changed in a5719bb369e7719c5b405b1ae4171e582c34b8cd.